packages/UI/src/components/organisms/Navigation/NotificationPopover/NotificationPopover.tsx

```diff
@@ -112,3 +119,6 @@ export default function NotificationPopover() {
     </Popover>
   );
 }
+
+const SettingsIcon = Settings as React.FC<React.SVGProps<SVGSVGElement>>;
```

Comment:
@yacineKahlerras what do you think of putting these lines at the top of the file, for readability?

Reply:
@dbradham yeah that works, its just type casting because typescript was complaining so i figured they didnt matter as much